app/views/distributions/index.html.erb

```diff
@@ -117,7 +117,7 @@
                 <th>Delivery method</th>
                 <th>Shipping Cost</th>
                 <th>Comments</th>
-                <th>State</th>
+                <th>Status</th>
```

Comment:
I agree that state is not the best term here -- and that it should be status throughout   I think, if we're going to change state to status here,  we also need to change it in some other places (e.g. export,  perhaps even the name of the variable, which seems to be state, if I'm  reading correctly)

How about removing the state to status change from this PR,  and we'll put up an issue soonest about changing state to status on the distribution throughout?  Perhaps including the name of the variable?

Reply:
Good point. I've reverted the change as you've suggested.